controller/linenotify.go

```diff
@@ -16,48 +15,49 @@ import (
 
 // LINE Notify と連携する
 func LinenotifyOn(w http.ResponseWriter, req *http.Request) {
-	ctx := app.GetContext(req)
+	ctx := req.Context()
 
 	c, err := linenotify.NewAuthorization(config.C.Linenotify.ClientID, config.C.App.BaseURL+"/linenotify/callback")
 	if err != nil {
-		ctx.Fail(err)
+		fail(ctx, w, err, http.StatusInternalServerError)
 		return
 	}
 	http.SetCookie(w, &http.Cookie{Name: "state", Value: c.State, Expires: time.Now().Add(300 * time.Second), Secure: true})
 
 	err = c.Redirect(w, req)
 	if err != nil {
-		ctx.Fail(err)
+		fail(ctx, w, err, http.StatusInternalServerError)
 		return
 	}
 }
 
 // LINE Notify の連携を解除する
 func LinenotifyOff(w http.ResponseWriter, req *http.Request) {
-	ctx := app.GetContext(req)
+	ctx := req.Context()
+
 	log.Info(ctx, "Redirect to LINE Notification revoke page")
 
 	// official url
 	http.Redirect(w, req, "https://notify-bot.line.me/my/", http.StatusFound)
 }
 
 func LinenotifyCallback(w http.ResponseWriter, req *http.Request) {
-	ctx := app.GetContext(req)
+	ctx := req.Context()
 
 	params, err := linenotify.ParseAuthorization(req)
 	if err != nil {
-		ctx.Fail(err)
+		fail(ctx, w, err, http.StatusInternalServerError)
 		return
 	}
 
 	state, err := req.Cookie("state")
 	if err != nil {
-		ctx.Fail(err)
+		fail(ctx, w, err, http.StatusInternalServerError)
 		return
 	}
 
 	if params.State != state.Value {
-		ctx.Error(errors.New("Invalid csrf token."), http.StatusBadRequest)
+		fail(ctx, w, errors.New("Invalid csrf token."), http.StatusBadRequest)
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
error strings should not be capitalized or end with punctuation or a newline